salt/modules/win_servermanager.py

```diff
@@ -57,11 +58,30 @@ def _pshell_json(cmd, cwd=None):
     if 'convertto-json' not in cmd.lower():
         cmd = '{0} | ConvertTo-Json'.format(cmd)
     log.debug('PowerShell: %s', cmd)
-    ret = __salt__['cmd.shell'](cmd, shell='powershell', cwd=cwd)
+    ret = __salt__['cmd.run_all'](cmd, shell='powershell', cwd=cwd)
+
+    ret.pop()
```

Comment:
Did this `ret.pop` slip in by mistake? If not, would you explain what this is for?

Reply:
You know it did!